index.html

```diff
@@ -1,115 +1,276 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Baystation 12</title>
+		<link rel="icon" href="/favicon.ico">
+		<style>
```

Comment:
Possibly also going to extract the js blob at the bottom into a `<script defer src="..."></>` since it's a little ugly to be tacked on there. Because you know, people read the source.